.gitignore

```diff
@@ -0,0 +1,24 @@
+# If you prefer the allow list template instead of the deny list, see community template:
```

Comment:
This is your personal configuration; it belongs in $HOME/.gitignore, not this project.

Reply:
Yes it's opinionated, I just copied it from https://github.com/github/gitignore/blob/main/Go.gitignore